ambari-server/src/main/java/org/apache/ambari/server/actionmanager/ExecutionCommandWrapper.java

```diff
@@ -240,9 +240,9 @@ public void setVersions(Cluster cluster, CommandRepository commandRepository) {
         }
 
         if (null != commandRepository && repositoryVersion.isResolved() &&
-          !repositoryVersion.getVersion().equals(commandRepository.getRepositoryVersion())) {
+          !repositoryVersion.getVersion().equals(commandRepository.getRepoVersion())) {
```

Comment:
Curious why the name change here?

Reply:
@ncole when i have removed repoVersion replacement i found that server with command sending 2 fields:  repoVersion and repositoryVersion. On debugging i found that with object and annotation everything is fine, but somewhy spring serialize it with double field. I assuming that the reason is in a getter name as no other reasons found. 

Now i have checked the cluster and it confirmed, after renaming getter Spring serialized the object properly